data_model/src/query.rs

```diff
@@ -130,35 +126,21 @@ declare_versioned_with_scale!(VersionedSignedQueryRequest 1..2, Debug, Clone, ir
 
 /// I/O ready structure to send queries.
 #[version_with_scale(n = 1, versioned = "VersionedSignedQueryRequest")]
-#[derive(Debug, Clone, Io, Decode, Encode, Deserialize, Serialize, IntoSchema)]
+#[derive(Debug, Clone, Decode, Encode, Deserialize, Serialize, IntoSchema)]
 pub struct SignedQueryRequest {
     /// Payload
     pub payload: Payload,
     /// Signature of the client who sends this query.
     pub signature: SignatureOf<Payload>,
 }
 
-declare_versioned_with_scale!(VersionedQueryResult 1..2, Debug, Clone, iroha_macro::FromVariant, Io, IntoSchema);
+declare_versioned_with_scale!(VersionedQueryResult 1..2, Debug, Clone, iroha_macro::FromVariant, IntoSchema);
 
 /// Sized container for all possible Query results.
 #[version_with_scale(n = 1, versioned = "VersionedQueryResult")]
-#[derive(Debug, Clone, Io, Serialize, Deserialize, Encode, Decode, IntoSchema)]
+#[derive(Debug, Clone, Decode, Encode, Deserialize, Serialize, IntoSchema)]
 pub struct QueryResult(pub Value);
 
-#[cfg(feature = "warp")]
-impl Reply for &QueryResult {
-    fn into_response(self) -> Response {
-        Response::new(Vec::from(self).into())
-    }
-}
-
-#[cfg(feature = "warp")]
-impl Reply for QueryResult {
-    fn into_response(self) -> Response {
-        (&self).into_response()
-    }
-}
```

Comment:
Was it moved somewhere?

Reply:
Or is it just unused in current torii impl?

Reply:
removed because it is unused. I think this relates to the change I did about using `VersionedQueryResult` instead of `QueryResult`

Reply:
Ok